### ISSUE TITLE
Repository error message if CRAN mirror not set

### ISSUE THREAD

**Issue by andrie** [RevolutionAnalytics/miniCRAN#66]
Issue flagged at https://github.com/RevolutionAnalytics/AzureML/issues/63 by @samuel100

---

Using the following repository in RRE 7.4.1 (CentOS 6.5):

```
> options("repos")
$repos
                                                     REVO 
"http://packages.revolutionanalytics.com/cran/3.1/stable" 
```

causes the following error: 

```
Created new folder: /tmp/RtmpFFSrSy/dir13f457fe439e/packages/bin/windows/contrib/3.1
Error in if (!is.null(names(repos)) && repos["CRAN"] == "@CRAN@") { : 
  missing value where TRUE/FALSE needed
```

**Comment by andrie** [comment]
This is reproducible using:

```r
library(miniCRAN)
r <- c(REVO = "http://packages.revolutionanalytics.com/cran/3.1/stable")
pkg <- "MASS"
if(!exists("td")) td <- tempdir()
makeRepo(path = td, pkgs = pkg, repos = r)
```

**Comment by andrie** [comment]
This is now fixed in the dev branch and will be released with v0.2.5
